templates/cloud/cloud-computing.html

```diff
@@ -0,0 +1,480 @@
+{% extends "cloud/_base-cloud.html" %}
+
+{% block title %}What is a cloud computing?{% endblock %}
+
+{% block meta_description %}Cloud computing is an on-demand delivery of IT resources over a network. The cloud resource provisioning process is fully automated, leading to increased agility and significant cost savings.{% endblock meta_description %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1oIggNSXR5sfl0Ab35x2NIL7I858I7QdPCUXeQI8IIPc/edit#{% endblock meta_copydoc %}
+
+{% block content %}
+<section class="p-strip--suru-cloud-computing">
+  <div class="row">
+    <div class="col-6">
+      <h1 class="p-heading--2">What is<br class="u-hide--small u-hide--medium" />cloud computing?</h1>
```

Comment:
There is a space missing on mobile :scream: 
![image](https://user-images.githubusercontent.com/2707508/223172926-6f4e9e21-076c-478a-96a8-7d525c95761f.png)